internal/services/servicebus/servicebus_queue_data_source.go

```diff
@@ -155,6 +176,7 @@ func dataSourceServiceBusQueueRead(d *pluginsdk.ResourceData, meta interface{})
 		resourceGroup = namespaceId.ResourceGroupName
 		namespaceName = namespaceId.NamespaceName
 	} else {
+		// TODO remove this else block in 5.0
```

Comment:
This wants to be wrapped in a 5.0 block instead of just a comment about removing in 5.0. Otherwise it'll panic in 5.0 as these attributes won't be in the schema